libsolidity/inlineasm/AsmAnalysis.h

```diff
@@ -61,11 +61,12 @@ struct AsmAnalysisInfo;
 class AsmAnalyzer: public boost::static_visitor<bool>
 {
 public:
-	AsmAnalyzer(
+	explicit AsmAnalyzer(
```

Comment:
`explicit` is only useful if there is a constructor that takes only one (required) argument. But it does not hurt either.